src/main/java/net/fabricmc/loader/gui/FabricGuiEntry.java

```diff
@@ -38,10 +42,14 @@ private static void openWindow(FabricStatusTree tree, boolean shouldWait) throws
 		FabricMainWindow.open(tree, shouldWait);
 	}
 
-	/** @param exitAfter If true then this will call {@link System#exit(int)} after showing the gui, otherwise this will
-	 *            return normally. */
-	public static void displayCriticalError(Throwable exception, boolean exitAfter) {
-		FabricLoader.INSTANCE.getLogger().fatal("A critical error occurred", exception);
+	/**
+	 * @param exitAfter If true then this will call {@link System#exit(int)} after showing the gui, otherwise this will
+	 *                  return normally.
+	 */
+	public static void displayCriticalErrors(Set<Throwable> exceptions, boolean exitAfter) {
+		for (Throwable t : exceptions) {
+			FabricLoader.INSTANCE.getLogger().fatal("A critical error occurred", t);
+		}
```

Comment:
Please don't change this - this might want be used in other places in the future, and those places might only need to show a single error. *(Although this goes hand-in-hand with the [above comment]())*.